opensrp-reveal/build.gradle

```diff
@@ -177,15 +177,15 @@ tasks.withType(Test) {
 
 dependencies {
 
-    implementation('io.ona.kujaku:library:0.6.1') {
+    implementation('io.ona.kujaku:library:0.6.2') {
```

Comment:
Publish the updated Kujaku on master to this version

Reply:
Done.